### PR TITLE
+doc Mobject.get_critical_point()

### PR DESCRIPTION
**Pull request by Gutza**
## Motivation

Minor documentation improvement

## Overview / Explanation for Changes

Just added a few details to a single function's documentation.

## Oneline Summary of Changes

```
- Added new feature ... / Fixed bug ... / ... (:pr:`PR NUMBER HERE`)
```
- Documentation improvement for Mobject.get_critical_point()

## Further Comments

I just wanted to test a minor doc improvement for now because I don't know how to test documentation generation locally on my computer before submitting. What is the recommended workflow for this?

## Acknowledgements
- [x] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)

## Reviewer Checklist
- [ ] Newly added functions/classes are either private or have a docstring
- [ ] Newly added functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] Newly added documentation builds, looks correctly formatted, and adds no additional build warnings
- [ ] The oneline summary has been included [in the wiki](https://github.com/ManimCommunity/manim/wiki/Changelog-for-next-release)

### CODE REVIEW

File: manim/mobject/mobject.py
Comment (jsonvillanueva) [inline manim/mobject/mobject.py:1189]:
```suggestion
        ::

            sample = Arc(start_angle=PI/7, angle = PI/5)
```

**Review by jsonvillanueva — CHANGES_REQUESTED** [review]
Hi @Gutza, thanks for the contribution! Unfortunately your usage of example python code is rendered improperly. I believe all that is required is a new line between the `::` and your given Python code. You may wish to run `make html` command in the `manim/docs` directory to build the documentation locally before adding further commits.

You may also view the docs generated by your PR [here](https://manimce--1077.org.readthedocs.build/en/1077/reference/manim.mobject.mobject.Mobject.html#manim.mobject.mobject.Mobject.get_critical_point).

File: manim/mobject/mobject.py
Comment (Gutza) [reply manim/mobject/mobject.py:1189]: Thank you, those are exactly the pointers I was looking for! I'll fix the commit later today--but most importantly, I will set up my system so I can actually test the documentation markup before submitting in the future. :)

Comment (jsonvillanueva) [reply manim/mobject/mobject.py:1189]: Hey @Gutza, just checking in to see how you're doing with the documentation. Let me know if you've run into any issues, or reach out to any one of us in the `#docs` channel over on our [discord](https://discord.com/channels/581738731934056449/589666750082187302).

Comment (Gutza) [reply manim/mobject/mobject.py:1189]: Hello, @jsonvillanueva, thank you for reaching out! I did try to join the discord channel, but no joy using that link ("_you find yourself in a strange place_"). I did encounter a surprisingly silly issue when trying to build the documentation myself: if I install manim (`pip3 install manim`), it builds the documentation for the public version (which is not fun, since I can't see what my changes look like); if I uninstall manim (`pip3 uninstall manim`) I receive an error (`Could not import extension manim_directive (exception: No module named 'manim')`). Catch-22.

Comment (Gutza) [reply manim/mobject/mobject.py:1189]: It turns out the issue was even sillier than we could've ever imagined: I never knew I had to execute `python3 -m pip install .` after checking out the git repo, per the [core dev help page](https://docs.manim.community/en/latest/installation/for_dev.html). Thanks to **aquabeam** on discord for pointing me in the right direction, and sorry for wasting everyone's time with this!

Comment (jsonvillanueva) [reply manim/mobject/mobject.py:1189]: Thanks for the warning on the discord Link. I sort of assumed that it if you weren't already part of the discord, it would take you to create an account and then redirect you to the `#docs` channel... and if you were logged in/already part of the Manim discord, it would just take you. Good to know. We can continue discussion specific to dev-installation there, but I'm sure most devs use [poetry](https://python-poetry.org/docs/#installation) in [their installations ](https://docs.manim.community/en/latest/installation/for_dev.html#manim-installation-for-developers-with-poetry)... this way you can do an editable install (in tandem with `git` version control)

**Comment by Gutza** [comment]
Thank you @jsonvillanueva (here), and friedkeenan (on discord) for your availability--especially since all of this effort was only related to really minor improvements to the documentation! I genuinely hope I will find the time to contribute in a meaningful way; you've been quite amazing!

File: manim/mobject/mobject.py
Comment (Gutza) [reply manim/mobject/mobject.py]: Sorry, I hadn't noticed your reply when I published my final comment. I think it would be wise for me to check out the _poetry_ approach before deciding on a way to move forward. Thank you again!

Comment (Gutza) [reply manim/mobject/mobject.py]: You were right, poetry does indeed offer a way more streamlined experience, that's certainly the way to work!

File: docs/source/installation/for_dev.rst
Comment (behackl) [inline docs/source/installation/for_dev.rst:23]:
```suggestion
Please see :doc:`our contributing guide<contributing>` for more details about contributing to Manim.
```

**Review by behackl — CHANGES_REQUESTED** [review]
There is just one broken link, I've tried to add a suggestion that might fix it -- but as soon as this is taken care of, this PR can be merged. Thank you for your work!

File: docs/source/installation/for_dev.rst
Comment (Gutza) [reply docs/source/installation/for_dev.rst]: I integrated your suggestion, although my change there was itself supposed to resolve a broken link. I'll investigate that further later on. Thank you!

File: docs/source/installation/for_dev.rst
Comment (behackl) [inline docs/source/installation/for_dev.rst:23]: This actually fixes the reference, just tested it locally. :-)

```suggestion
Please see :doc:`our contributing guide<../contributing>` for more details about contributing to Manim.
```

**Review by behackl — APPROVED** [review]

**Review by jsonvillanueva — APPROVED** [review]

**Pull request merged**
